exampleSite/content/_index/logos-header.md

```diff
@@ -0,0 +1,9 @@
++++
+date = "2018-07-13"
+fragment = "section-header"
```

Comment:
Maybe we can simplify it to one word: `header`?

Reply:
I was thinking maybe in the future we may have another fragment for the website header. I'm not sure though.

Reply:
I was thinking maybe in the future we may have another fragment for the website header. I'm not sure though.

Reply:
Ok then let's keep section-header for now